android-o/AutofillFramework/AutofillFramework/multidatasetservice/settings/MyPreferences.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using Android.Content;
+
+namespace AutofillFramework.multidatasetservice.settings
+{
+	public class MyPreferences
+	{
+		static String TAG = "MyPreferences";
```

Comment:
This appears to be unused. Though it does exist in the Java sample I think it can safely be removed.

Reply:
Done